src/main/java/picard/fingerprint/FingerprintMetrics.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2019 The Broad Institute
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package picard.fingerprint;
+
+import htsjdk.samtools.metrics.MetricBase;
+import picard.analysis.FingerprintingDetailMetrics;
+import picard.analysis.FingerprintingSummaryMetrics;
+
+/**
+ * Class for holding metrics on a single fingerprint.
+ * Note: this is distinct from {@link FingerprintingDetailMetrics} and
+ * {@link FingerprintingSummaryMetrics} in that it is calculated on a single fingerprint,
+ * and attempts to describe how likely that fingerprint is to have arisen from an actual sample
+ * as opposed to having artifacts such as contamination, or strong bias towards homozygous genotypes.
+ */
+public class FingerprintMetrics extends MetricBase {
+    /* The Sample alias taken from RG header or #CROME line */
+    public String SAMPLE_ALIAS;
+
+    /* The originating file (if available) for this fingerprint */
+    public String SOURCE;
+
+    /* Additional information about the fingerprint */
+    public String INFO;
+
+    /* Number of haplotypes examined */
+    public long HAPLOTYPES;
+
+    /* Number of haplotypes that had evidence in the source file */
+    public long HAPLOTYPES_WITH_EVIDENCE;
+
+    /* Number of haplotypes that had enough evidence to make a definite genotype call */
+    public long DEFINITE_GENOTYPES;
+
+    /* Number of major allele homozygous calls*/
+    public long NUM_HOM_ALLELE1;
+
+    /* Number of definite heterozygous calls*/
```

Comment:
What does definite mean here? These are the number of het calls (that haven't been filtered by `GENOTYPE_LOD_THRESHOLD`) so this is different from `DEFINITE_GENOTYPES`, is that right? (And same for `NUM_HOM_ALLELE2 below`)

Reply:
same.... DEFINITE_GENOTYPES= NUM_HOM_ALLELE2 + NUM_HOM_ALLELE1 + NUM_HET

Reply:
actually, that's not true...I'll make a better note of it.

Reply:
I still don't understand this distinction. It looks like `NUM_HET`, `NUM_HOM_ALLELE1`, and `NUM_HOM_ALLELE2` are not "definite" because they aren't filtered by `GENOTYPE_LOD_THRESHOLD`. I would either remove the word definite here, or make it clearer [here](https://github.com/broadinstitute/picard/blob/yf_add_pvalue_to_fingerprinting/src/main/java/picard/fingerprint/CalculateFingerprintMetrics.java#L169) why these metrics are "definite" calls rather than just calls.